app/services/direct-payments/create-payment-file.js

```diff
@@ -29,6 +29,6 @@ function mkdirIfNotExists (dir) {
 }
 
 function getFileName () {
-  const datestamp = dateFormatter.now().format('YYYYMMDD')
+  const datestamp = dateFormatter.now().format('YYYYMMDDhhmmss')
```

Comment:
Ran a payment run at 1618 and got this name, `apvs-payments-20170130041803.csv`. I think moment uses a different format for 24h time.